Lib/test/test_asyncio/test_tasks.py

```diff
@@ -4,6 +4,7 @@
 import contextlib
 import functools
 import io
+import gc
```

Comment:
I agree, good idea. BTW help is always welcome by python/core-workflow. You're an awesome contributor, we'd love you to continue! :)